### PR TITLE
Fixed typo in tests/admin_docs/models.py.

### PR DESCRIPTION
**Pull request by davidolz**
Fixed typo in django/tests/admin_docs/models.py

### CODE REVIEW

**Review by shangxiao — APPROVED** [review]
🏆 Nice catch

File: django/db/models/aggregates.py [hunk 1]
```diff
@@ -53,9 +53,9 @@ def get_source_expressions(self):
             return source_expressions + [self.filter]
         return source_expressions
 
-    def set_source_expressions(self, exprs):
-        self.filter = self.filter and exprs.pop()
-        return super().set_source_expressions(exprs)
+    def set_source_expressions(self, expression):
+        self.filter = self.filter and expression.pop()
+        return super().set_source_expressions(expression)
```
Comment (felixxm) [inline django/db/models/aggregates.py:58]: This is not a typo, please revert.

Comment (shangxiao) [reply django/db/models/aggregates.py:58]: Also this was pushed after I reviewed the changes, that's not ok in my book 😛

Comment (davidolz) [reply django/db/models/aggregates.py]: I am sorry, my mistake. Changes reverted 😎

**Comment by felixxm** [comment]
@davidolz Thanks :+1: Welcome aboard :boat:

**Review by felixxm — APPROVED** [review]

**Pull request merged**